src/main/java/org/jenkinsci/plugins/plaincredentials/impl/FileCredentialsImpl.java

```diff
@@ -229,6 +220,29 @@ public InputStream getContent() throws IOException {
         return new ByteArrayInputStream(secretBytes.getPlainData());
     }
 
+    private Optional<FileItem> resolveFile(FileItem file) {
+        return Optional.ofNullable(file).filter(f -> f.getName().length() > 0);
+    }
+
+    private SecretBytes resolveSecretBytes(FileItem file, SecretBytes secretBytes) {
+        SecretBytes bytes = Optional.ofNullable(secretBytes).orElseThrow(() -> new IllegalArgumentException("No content provided or resolved."));
+        return resolveFile(file)
+                .map(f -> SecretBytes.fromBytes(f.get()))
+                .orElse(bytes);
+    }
+
+    private SecretBytes resolveSecretBytes(FileItem file, String data) {
```

Comment:
Since both overloads are only called once, could they just be inlined back to where they were? It is hard to scroll back and forth to see what these methods are doing.